app/views/performance_dashboards/project_type/index_pdf.haml

```diff
@@ -11,7 +11,7 @@
   :javascript
     $(function() {
       'use strict';
-      var opts = {};
+      var opts = { truncate_labels: 50 };
```

Comment:
Should the HTML and PDF truncations be the same, or are they deliberately different?

Reply:
Yes, the font sizes differ, so we can fit more words in the HTML version.